mobile/js_files/package.json

```diff
@@ -38,7 +38,7 @@
     "react-native-screens": "^1.0.0-alpha.23",
     "react-native-shake": "^3.3.1",
     "react-native-splash-screen": "^3.2.0",
-    "react-native-status-keycard": "^2.5.14",
+    "react-native-status-keycard": "git+https://github.com/status-im/react-native-status-keycard.git#v2.5.16",
```

Comment:
why not just "^2.5.16" ?

Reply:
Don't know which credentials to use to upload to NPM. I'll use the git url for now if no objections.